config/jetpack-cloud-development.json

```diff
@@ -44,6 +44,7 @@
 		"layout/query-selected-editor": false,
 		"layout/support-article-dialog": false,
 		"oauth": true,
+		"persist-redux": true,
```

Comment:
I'd consider removing the `persist-redux` feature flag completely, and assume it's always `true`.

The flag was probably added at a time when Redux persistence was new and risky. But now it's been enabled for many years, and became boring 🙂

Reply:
That's a great idea. There is no need to have it if it's enabled in every environment. I'll clean that up.